src/pkg/packaging/dir.props

```diff
@@ -28,6 +30,7 @@
     <ProductBrandSuffix Condition="'$(ReleaseBrandSuffix)'!=''">$(ProductionVersion) $(ReleaseBrandSuffix)</ProductBrandSuffix>
     <SharedHostBrandName>$(ProductBrandPrefix) Host - $(ProductBrandSuffix)</SharedHostBrandName>
     <HostFxrBrandName>$(ProductBrandPrefix) Host FX Resolver - $(ProductBrandSuffix)</HostFxrBrandName>
+    <TargetingPackBrandName>$(ProductBrandPrefix) App Targeting Pack - $(ProductBrandSuffix)</TargetingPackBrandName>
```

Comment:
"App Targeting Pack" is odd branding. I suggest removing App.

Reply:
This expands to `Microsoft .NET Core App Targeting Pack`, is `Microsoft .NET Core Targeting Pack` actually specific enough?

Reply:
I think that's good branding for the base targeting pack (without app). The others will be ASP.NET Core and Windows Desktop to distinguish form .NETCore. The App name in the packages is unfortunate in my opinion. I think it goes back to .NETCore being taken in TFMs before what we now call .NET Core being .NETCoreApp -> NETCore.App. It's a weird name because you can use it in libraries IMHO.

I'm haven't seen ".NET Core App" before in branding.

Reply:
I see, my view of the product has previously always been in nuget packages, so .NET Core App looks ordinary to me. Thanks for the details, will change.